src/components/CreateCoursePage/index.jsx

```diff
@@ -39,6 +40,15 @@ class CreateCoursePage extends React.Component {
     this.setState({ startedFetching: true });
   }
 
+  isOrgWhitelisted() {
+    const userOrgs = this.props.publisherUserInfo.organizations;
+    if (typeWhiteList.length > 0 && typeWhiteList[0] === '-') {
+      const typeBlackList = typeWhiteList.slice(1);
+      return userOrgs.some(org => !typeBlackList.includes(org.key));
```

Comment:
This means that a multi-org person would be allowed to use types if any of their orgs are not blacklisted. Maybe it should be if like `!userOrgs.any(org => typeBlackList.includes`? I dunno, multi-org people are a weird case.

Reply:
Yeah, I was thinking about that. I don't know what the right way of doing it is. I think the way I have it is the most correct because it at least mirrors the behavior of the whitelist. Like right now with the whitelist, as soon as one of your orgs is in the whitelist you can see it. Or I update both to `every`?